audit/src/main/scala/tech/beshu/ror/audit/instances/DefaultAuditLogSerializerV1.scala

```diff
@@ -17,82 +17,35 @@
 package tech.beshu.ror.audit.instances
 
 import org.json.JSONObject
-import tech.beshu.ror.audit.AuditResponseContext._
-import tech.beshu.ror.audit.{AuditLogSerializer, AuditRequestContext, AuditResponseContext}
-
-import java.time.ZoneId
-import java.time.format.DateTimeFormatter
-import scala.collection.JavaConverters._
-import scala.concurrent.duration.FiniteDuration
+import tech.beshu.ror.audit._
+import tech.beshu.ror.audit.AuditResponseContext.Verbosity
+import tech.beshu.ror.audit.utils.AuditSerializationHelper
+import tech.beshu.ror.audit.utils.AuditSerializationHelper.AllowedEventMode.Include
+import tech.beshu.ror.audit.utils.AuditSerializationHelper.AuditFieldGroup.CommonFields
+
+/**
+ * Serializer for audit events (V1) that is aware of **rule-defined verbosity**.
+ *
+ * - Includes only `CommonFields` (request metadata, user, action, etc.).
+ * - Serializes all non-Allowed events.
+ * - Serializes `Allowed` events only if the corresponding rule
+ * specifies that they should be logged at `Verbosity.Info`.
+ *
+ * Recommended when a minimal, compact audit log is sufficient.
+ */
+class BlockVerbosityAwareAuditLogSerializerV1 extends DefaultAuditLogSerializerV1
```

Comment:
I don't get why we introduce V1 and V2 for the "new" serializer (to be strict - renamed). 

I took into consideration two things:
1. backward compatibility - we don't have to do that
2. documentation - how are we going to explain to our users (new users, and users who have ROR already installed) in simple words the V1 and V2? - so it may be misleading for them. Do they need it? I suppose - no.